.github/workflows/common.yml

```diff
@@ -3,7 +3,11 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'python/**'
```

Comment:
Consider adding `'*.py'` to the path filters to catch Python files in the root directory. Currently, only files under the `python/` directory will trigger the workflows.